db/src/commonMain/sqldelight/com/islamversity/db/tables.sq

```diff
@@ -0,0 +1,101 @@
+import com.islamversity.db.model.EntityUUID;
+import com.islamversity.db.model.Calligraphy;
+
+CREATE TABLE IF NOT EXISTS calligraphy (
+    index INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    id TEXT AS EntityUUID NOT NULL ,
+    code TEXT AS Calligraphy UNIQUE NOT NULL
+);
+
+CREATE UNIQUE INDEX IF NOT EXISTS calligraphy_id
+ON calligraphy (id);
+
+CREATE UNIQUE INDEX IF NOT EXISTS calligraphy_code
+ON calligraphy (code);
+
+CREATE VIEW IF NOT EXISTS no_index_calligraphy
+AS
+SELECT id, code FROM calligraphy;
+
+CREATE TABLE IF NOT EXISTS name (
+    index INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    id TEXT AS EntityUUID NOT NULL,
+    parentId TEXT AS EntityUUID NOT NULL,
```

Comment:
need relation from parentId to soraId

Reply:
I'm not sure how we can do that. please send your ideas.
the reason for the uncertainty is,we said: "we can use `name` column for other tables like books.name as well"
not sure how I can create relations for all of them